packages/components/src/components/dropdown/dropdown.stories.ts

```diff
@@ -101,260 +115,149 @@ export const Default = {
 };
 
 /**
- * For round triggers (like e. g. default `sd-buttons`), set the `rounded` attribute. This sets a rounding on the dropdown and automatically adds a minimal distance of 1px between the trigger and the panel.
+ * Use the `open` attribute to toggle the visibility of the dropdown.
+ */
+
+export const Open = {
+  name: 'Open',
+  render: () => html`
+    <sd-dropdown open>
+      <div class="slot slot--border slot--background w-52 h-12">Default Slot</div>
+      <sd-button slot="trigger">Trigger</sd-button>
+    </sd-dropdown>
+  `
+};
+
+/**
+ * Use the `rounded` attribute to set a border-radius on the dropdown panel and trigger button.
  */
 
 export const Rounded = {
-  parameters: { controls: { exclude: ['rounded', 'default'] } },
-  render: (args: any) =>
-    generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'rounded'
-        }
-      },
-      constants: {
-        type: 'slot',
-        name: 'default',
-        value: '<div class="example"></div>'
-      },
-      args
-    })
+  name: 'Rounded',
+  render: () => html`
+    <sd-dropdown open rounded>
+      <div class="slot slot--border slot--background w-52 h-12">Default Slot</div>
+      <sd-button slot="trigger">Trigger</sd-button>
```

Comment:
Use sd-button/secondary as icon-only displaying system icon "more-functions"

See [updated figma ](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2357-10943&t=UpPGX0jehyNFfgNq-11)

[more-functions](https://www.figma.com/design/KNIe24a9qGUmpZ4ntnPN5P/Solid-DS-%E2%80%93-Icon-Asset-Library?node-id=1335-2305&t=x1YbFe33DH3gfBi8-11)

Reply:
And set the distance between trigger and dropdown to 4px

Reply:
Updated.